libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss

```diff
@@ -100,12 +82,12 @@ pds-tab {
 
   .pds-tab {
     background-color: var(--color-background-tab);
-    border-radius: var(--border-radius-availability) var(--border-radius-availability) var(--border-radius-none) var(--border-radius-none);
+    border-radius: calc(var(--pine-dimension-xs) * 1.25) calc(var(--pine-dimension-xs) * 1.25) var(--pine-dimension-none) var(--pine-dimension-none);
```

Comment:
I know we discussed this, but let's revisit once we are done. For the calculations, I think we should hard code the expected value, just because if values change it could have a larger impact.